apps/src/sites/studio/pages/projects/index.js

```diff
@@ -67,12 +68,22 @@ $(document).ready(() => {
   });
 
   if (experiments.isEnabled(experiments.REACT_PROJECTS_TABLE)) {
-    ReactDOM.render(
-      <PersonalProjectsTable
-        projectList={[]}
-      />,
-    document.getElementById('react-my-projects')
-    );
+    const personalProjectsUrl = `/api/v1/projects/personal/${projectsData.userId}`;
+
+    $.ajax({
+      method: 'GET',
+      url: personalProjectsUrl,
+      dataType: 'json'
+    }).done(personalProjectsList => {
+      store.dispatch(setPersonalProjectsList(personalProjectsList));
+
+      ReactDOM.render(
+        <Provider store={store}>
+          <PersonalProjectsTable/>
+        </Provider>,
+       document.getElementById('react-my-projects')
+      );
+    });
```

Comment:
@aoby this is definitely a pattern we use in other places. Within star labs / teacher features, I believe we tend to do it this way, unless there is a particularly slow page or other UX problem during load. the approach you describe would then be my first choice to solve a problem with a page that looks broken while loading and/or is slow to load. I could be convinced that rendering statically (an empty projects table in this case) is a good best practice, although I believe @breville  has some allergies / pushback to the loading spinners that often come along with these. This could be a good discussion for the [#design](https://codedotorg.slack.com/messages/C0SUN2SSF/) room too.

Reply:
There's no reason it needs to display a spinner. It could render `null` in the empty case. The spinner / no spinner debate is separate from the static / dynamic React render. 

I don't feel too strongly about it if this pattern is used elsewhere. I just haven't seen it. I just find React classes to be really easy to reason about. If it's always present, the state determines what is displayed. If it's sometimes present based on an API callback, that might be harder to debug or decipher?

Reply:
I don't feel strongly about this either way, but a minor benefit of rendering the component in the outer ("static") context might be that it becomes easier to attach to a parent React component later as we gradually render more of our page with React.  I'm not sure that will matter in this particular case though.

Reply:
OK, in the absence of strong opposition let's leave this how it is then. In the future maybe we can standardize on a best practice with input from UX/PM.